CRM/Event/Form/SelfSvcTransfer.php

```diff
@@ -522,7 +518,7 @@ public function transferParticipantRegistration($toContactID, $fromParticipantID
       $value_to['fee_amount'] = $dao->fee_amount;
     }
     $value_to['contact_id'] = $toContactID;
-    $value_to['event_id'] = $eventID;
+    $value_to['event_id'] = $this->_event_id;
```

Comment:
given the above query we should probably just get event_id in the query

Reply:
done. Have updated with the api4 call and created the toParticipant with the retrieved params 👍